File: src/main/java/edu/marmara/service/AdvisorService.java
package edu.marmara.service;

import edu.marmara.model.Advisor;
import edu.marmara.model.Student;

import java.io.IOException;

public interface AdvisorService {
    Student getStudent(Long studentID, Advisor advisor);

    void approveSchedule(Student student) throws IOException;
}

File: src/main/marmara/service/AdvisorService.py
from src.main.marmara.model.Advisor import Advisor
from src.main.marmara.model.Student import Student


class AdvisorService:
    def get_student(self, student_id: int, advisor: Advisor) -> Student:
        pass

    def approve_schedule(self, student: Student) -> None:
        pass

    def deny_schedule(self, student: Student) -> None:
        pass
